### PR TITLE
Moar Py3 fixes

### PR DESCRIPTION
**Pull request by s0undt3ch**

### CODE REVIEW

File: tests/unit/modules/docker_test.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def test_ps_with_filters(self):
                 all=True,
                 filters={'label': 'KEY'})
 
-    @skipIf(_docker_py_version() > 0, 'docker-py needs to be installed for this test to run')
+    @skipIf(_docker_py_version() > (0,), 'docker-py needs to be installed for this test to run')
```
Comment (cachedout) [inline tests/unit/modules/docker_test.py]: Could you please confirm that the Docker test failure isn't related to this change?

Comment (s0undt3ch) [inline tests/unit/modules/docker_test.py]: Dam, I rebased and lost the jenkins build links.

Lets see what Jenkins says now.

Comment (s0undt3ch) [inline tests/unit/modules/docker_test.py]: So, the version comparison was in fact wrong.

**Comment by s0undt3ch** [comment]
Issues resolved. Merging.

**Pull request merged**